### PR TITLE
fix: changed max colony objective description length, change title to…

### PR DESCRIPTION
**Pull request by CzarekDryl**
https://tw.auroracreation.com/app/tasks/15675939
https://tw.auroracreation.com/app/tasks/15702165
https://tw.auroracreation.com/app/tasks/15695338

Changes in this PR:
- changed colony objective max description length
- changed title from textarea to input to prevent breaking
- all validation errors are now in one banner under form

### CODE REVIEW

File: src/components/v5/common/ObjectiveBox/ObjectiveBox.tsx [hunk 1]
```diff
@@ -27,7 +29,12 @@ const ObjectiveBox: FC<ObjectiveBoxProps> = ({ objective }) => {
         {objective?.title || formatMessage(MSG.noObjectiveBoxTitle)}
       </h6>
       <p className="text-sm text-gray-600 mb-[1.125rem]">
-        {objective?.description || formatMessage(MSG.noObjectiveBoxDescription)}
+        {(objective &&
```
Comment (bassgeta) [inline src/components/v5/common/ObjectiveBox/ObjectiveBox.tsx:32]: maybe we can rather use a ternary here

**Review by bassgeta — CHANGES_REQUESTED** [review]
Code looks good, just a small change to ternary and it should be good

File: src/components/v5/common/ObjectiveBox/ObjectiveBox.tsx (hunk 1 above)
Comment (CzarekDryl) [reply src/components/v5/common/ObjectiveBox/ObjectiveBox.tsx]: Ye you are right, I've used ternary here

**Review by bassgeta — APPROVED** [review]

**Comment by jakubcolony** [comment]
@CzarekDryl looks like this needs a rebase

**Comment by CzarekDryl** [comment]
@jakubcolony Rebase done

**Review by bassgeta — DISMISSED** [review]

**Review by rdig — APPROVED** [review]

**Comment by rdig** [comment]
Please write more descriptive and succinct PR titles, as the current format is longer than Github can display

**Comment by rdig** [comment]
This also needs another rebase and for the merge conflicts to be solved

**Comment by rdig** [comment]
This needs rebasing once again, since after #1540 merged it can no longer be tested

**Review by rdig — DISMISSED** [review]
Seems be working as expected:

Screenshot from 2024-01-04 12-39-37
Screenshot from 2024-01-04 12-40-06

Can we do the same, and limit the content length for the colony description as well?

Screenshot from 2024-01-04 12-39-47

**Review by ArmandoGraterol — DISMISSED** [review]
LGTM 👍

**Review by ArmandoGraterol — APPROVED** [review]

**Review by rdig — APPROVED** [review]
All good again

**Pull request merged**